go.mod

```diff
@@ -59,6 +59,7 @@ require (
 	google.golang.org/protobuf v1.33.0
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
 	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
yeah thats weird to me

Reply:
now we need to do a go mod tidy on it manually - ill just close this an update it as part of my routine updates

Reply:
It cannot replace `gopkg.in/yaml.v2` because the imports need to be updated in the code to `gopkg.in/yaml.v3`. But what I found weird is that it moved the dependency into direct dependencies.